Resources/config/mongodb.xml

```diff
@@ -210,5 +211,9 @@
 
         <!-- listeners -->
         <service id="doctrine_mongodb.odm.listeners.resolve_target_document" class="%doctrine_mongodb.odm.listeners.resolve_target_document.class%" public="false" />
+
+        <service id="doctrine.mongodb.odm.symfony.fixtures.loader" class="Doctrine\Bundle\MongoDBBundle\Loader\SymfonyFixturesLoader" public="false">
```

Comment:
Other services are named `doctrine_mongodb.odm...`. Please follow that convention.